components/Subheader.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+export default function Subheader({ selectedCategory }) {
+  if (!selectedCategory) return null;
+
+  const iconSrc = `/navigation-icons/${selectedCategory}.svg`;
+
+  return (
+    <SubheaderWrapper>
+      <CategoryIcon src={iconSrc} alt={selectedCategory} />
+      <CategoryTitle>{selectedCategory}</CategoryTitle>
+    </SubheaderWrapper>
+  );
+}
+
+const SubheaderWrapper = styled.header`
+  display: flex;
+  align-items: center;
+  padding: 0, 5rem;
```

Comment:
Das Komma kann weg. ;)

```suggestion
  padding: 0 5rem;
```